app/styles/app.scss

```diff
@@ -0,0 +1,22 @@
+@import "bootstrap";
+
+html, body {
+  background-color: #343333;
+  margin: 10px;
+  color: white;
+}
+
+#title {
+  text-align: center;
+}
+
+#skill-logo {
+  margin-top: 0px;
+  text-align: center;
+  font-family: "desdemona";
```

Comment:
I would imagine not, but why would we ever support anyone who doesn't have desdemona installed!?